src/components/pages/ChildDashboard/RenderChildDashboard.js

```diff
@@ -1,54 +1,58 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { Header } from '../../common';
 import ChildFooter from '../../common/ChildFooter';
 import { Row, Col } from 'antd';
 import { useHistory } from 'react-router-dom';
 import { InstructionsModal } from '../../common';
 import { modalInstructions } from '../../../utils/helpers';
-import { HUD } from '../HeadsUpDisplay/index';
-import Explosion from '../../../assets/images/gamemodeimg/explosion.png';
 import change_your_avatar from '../../../assets/images/child_dashboard_images/change_your_avatar.svg';
 import leaderboard_icon from '../../../assets/images/child_dashboard_images/leaderboard_icon.png';
+import data from '../../../data.json';
+import { connect } from 'react-redux';
 import { getSubmissions } from '../../../api/index';
 
 const RenderChildDashboard = props => {
-  const { push } = useHistory();
+  const history = useHistory();
   const [modalVisible, setModalVisible] = useState(false);
-  const [submissionData, setSubmissionData] = useState([]);
 
-  // ChildId hardcoded for now until global state of child is retrived
-  // Retrieves array of submissions pertaining to child id, response is in the form of an array
+  const [submissionData, setSubmissionData] = useState([]);
   useEffect(() => {
     getSubmissions(1).then(res => {
-      setSubmissionData(res);
+      console.groupCollapsed('%cgetSubmissions', 'color: #CC5500');
+      setSubmissionData(res.data);
+      console.log(
+        `%cgetSubmissions response: ${JSON.stringify(res)}`,
+        'color: #2980B9'
+      );
+      console.groupEnd('getSubmissions');
```

Comment:
Not sure if console statements can be pushed to prod, if they can, pls ignore this feedback :)

Reply:
I thought about that myself after submitting it. I chose to leave it because it is related to another card and will be useful when working on that card.

Reply:
I also figured that if I was going to leave a console.log, it would be well-formatted, easy to read and provide useful information.

Reply:
Thanks for clarification!